partner_financial_risk/models/res_partner.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from openerp import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    eval_risk_sale_order = fields.Boolean(
```

Comment:
About the naming, I'd make 2 suggestions: 

1. Keep all fields grouped by prefix. For example:
  - Put all under `risk_` prefix.
  - Group by model: `risk_sale_order_*`.
2. These `eval` fields... It is a bit misleading